pkg/apis/cluster/v1alpha1/machine_types.go

```diff
@@ -141,6 +141,14 @@ type MachineStatus struct {
 	// Addresses is a list of addresses assigned to the machine. Queried from cloud provider, if available.
 	// +optional
 	Addresses []corev1.NodeAddress `json:"addresses,omitempty"`
+
+	// List of conditions synced from the node-conditions of the corresponding node-object.
+	// Machine-controller would be responsible for keeping conditions up-to-date.
```

Comment:
nit: Machine controller is responsible ...

Reply:
fixed